sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -80,6 +80,20 @@ export class Sender {
     return sender.send(message);
   }
 
+  /**
+   * Opens the AMQP link to Azure Service Bus from the sender.
+   *
+   * It is not necessary to call this method in order to use the sender. It is
+   * recommended to call this before your first send() or sendBatch() call if you
+   * want to front load the work of setting up the AMQP link to the service.
+   */
+  open() {
```

Comment:
Since this is a public method and part of the API, can you add the return type? That way TypeScript will yell at us if we try to change our public API surface.

Reply:
Absolutely, that was not an intentional omission!